bin/start-release

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Start Maiko release from maiko/bin
+#     startrelease [tag]
+# tag defaults to maiko-YYMMDD
+
+case `gh version` in
```

Comment:
Should probably have the same check as release-one for "gh" before it tries to use it.